src/content/blog/ddev-docker-chromebook.md

```diff
@@ -12,49 +12,64 @@ categories:
   - Guides
 ---
 
-[DDEV](http://github.com/ddev/ddev) and Docker work fine for local development on an inexpensive Chromebook!
+## Introduction
 
-I’d been wanting to try out a Chromebook for DDEV development for a while and I confess that I did an impulse buy on Amazon and got the 8GB [Asus C425](https://www.asus.com/us/Laptops/ASUS-Chromebook-14-C425TA/) for $329, which is a lot less than the fancy loaded MacBook Pro I usually use. It works great, mostly because it runs pure Debian Linux in what it calls the “Terminal”.
+[DDEV](http://github.com/ddev/ddev) and Docker CE work fine for local development on a Chromebook!
 
-Here’s the step-by-step to set up a DDEV development environment:
+A few years ago I wrote the first version of this article after experimenting with an 8GB [Asus C425](https://www.asus.com/us/Laptops/ASUS-Chromebook-14-C425TA/) for $329, which is a lot less any fancy Mac. It worked great, mostly because Chromebooks can run pure Debian Linux in what they call the “Terminal”.
```

Comment:
```suggestion
A few years ago I wrote the first version of this article after experimenting with an 8GB [Asus C425](https://www.asus.com/us/Laptops/ASUS-Chromebook-14-C425TA/) for $329, which is a lot less than any fancy Mac. It worked great, mostly because Chromebooks can run pure Debian Linux in what they call the “Terminal”.
```